### PR TITLE
This PR does 3 things:

### PR DESCRIPTION
**Pull request by qidonna**
- use log.js instead of console.log, 
- using console.* will trigger failures instead of warnings
- exclude log.js and log-test.js from lint

### CODE REVIEW

File: src/runtime/client-event-manager.js [hunk 1]
```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
+/* eslint-disable */
```
Comment (mborof) [inline src/runtime/client-event-manager.js]: Why is this necessary?

Comment (qidonna) [reply src/runtime/client-event-manager.js]: I'll remove it. It was added by mistake. I deleted before pushing, but the file was not automatically saved. Good catch!
